stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/attachment/factory/camera/internal/CameraAttachmentFragment.kt

```diff
@@ -14,26 +14,29 @@
  * limitations under the License.
  */
 
-package io.getstream.chat.android.ui.message.input.attachment.camera.internal
+package io.getstream.chat.android.ui.message.input.attachment.factory.camera.internal
 
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.activity.result.ActivityResultLauncher
-import androidx.core.os.bundleOf
 import androidx.core.view.isVisible
 import androidx.fragment.app.Fragment
-import androidx.fragment.app.setFragmentResult
 import com.getstream.sdk.chat.CaptureMediaContract
 import com.getstream.sdk.chat.model.AttachmentMetaData
 import com.getstream.sdk.chat.utils.PermissionChecker
 import io.getstream.chat.android.ui.common.extensions.internal.streamThemeInflater
+import io.getstream.chat.android.ui.common.style.setTextStyle
 import io.getstream.chat.android.ui.databinding.StreamUiFragmentAttachmentCameraBinding
-import io.getstream.chat.android.ui.message.input.attachment.AttachmentSelectionDialogFragment
-import io.getstream.chat.android.ui.message.input.attachment.AttachmentSelectionDialogStyle
+import io.getstream.chat.android.ui.message.input.MessageInputViewStyle
+import io.getstream.chat.android.ui.message.input.attachment.AttachmentSource
+import io.getstream.chat.android.ui.message.input.attachment.factory.AttachmentsPickerTabListener
 import java.io.File
 
+/**
+ * Represents the third tab of the attachment picker with media capture.
```

Comment:
We should never add things like ordering or numbering in docs. In case this moves away from the "third" slot, it'll be incorrect.

`Represents the tab of the attachment picker with media capture.` is good